src/SFA.DAS.LevyTransferMatching.Web/Controllers/FindBusinessController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Controllers
+{
+    [AllowAnonymous]
+    public class FindBusinessController : Controller

```

Comment:
As I understand it, this is the `/` home page. I'd have been tempted then to call the Controller `HomeController`